ariadne_graphql_modules/next/uniontype.py

```diff
@@ -132,6 +129,21 @@ def validate_union_type_with_schema(cls: Type[GraphQLUnion]) -> NoReturn:
     validate_name(cls, definition)
     validate_description(cls, definition)
 
+    schema_type_names = {type_node.name.value for type_node in definition.types}
+
+    class_type_names = {t.__get_graphql_name__() for t in cls.__types__}
+    if not class_type_names.issubset(schema_type_names):
+        missing_in_schema = class_type_names - schema_type_names
+        raise ValueError(
+            f"Types {missing_in_schema} are defined in __types__ but not present in the __schema__."
```

Comment:
Wrap attributes names in `'__attrname__'` in error messages.

Reply:
Done

Reply:
This also applies `__types__` and `__schema__` and other Python or GraphQL names